src/Symfony/Component/Security/Core/Validator/Constraints/UserPassword.php

```diff
@@ -14,7 +14,10 @@
 use Symfony\Component\Validator\Constraint;
 
 /**
+ * An attribute to check that the target isn't equal to the current user password.
```

Comment:
I wonder if this description could be outdated "soon" if we add more logic to this attribute.

Maybe we could use a more generic description like: _"Performs some validations on user passwords."_

But ... given that this is a constraint called "user password" ... maybe that's the typical "no-description description". You are describing what you can immediately infer from the name of this thing. So, maybe it's better to not add a description of the attribute?